server/build/plugins/unlink-file-plugin.js

```diff
@@ -9,7 +9,7 @@ export default class UnlinkFilePlugin {
   apply (compiler) {
     compiler.plugin('after-emit', (compilation, callback) => {
       const removed = Object.keys(this.prevAssets)
-      .filter((a) => !compilation.assets[a])
+      .filter((a = '') => a.includes('bundles/pages/') && !compilation.assets[a])
```

Comment:
https://github.com/zeit/next.js/blob/53a2c5a7fc14dd7b6a32ed27080534eefd2362f8/server/utils.js#L4

Reply:
Didn't know this regexp exsits. I'll use this regexp instead 😏